pkg/kubelet/kubelet.go

```diff
@@ -3095,6 +3095,8 @@ func (kl *Kubelet) isContainerRuntimeVersionCompatible() error {
 		if result < 0 {
 			return fmt.Errorf("container runtime version is older than %s", dockertools.MinimumDockerAPIVersion)
 		}
+	case "rkt":
+		return kl.GetRuntime().(*rkt.Runtime).CheckVersion(rkt.MinimumRktBinVersion, rkt.RecommendedRktBinVersion, rkt.MinimumAppcVersion, rkt.MinimumRktApiVersion, rkt.MinimumSystemdVersion)
```

Comment:
@vishh Are you ok with pushing the version checking logic down to the runtime? It seems that you reviewed related PR before. :)